lib/core_ext/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -34,10 +34,11 @@ def add_index(table_name, column_name, options = {})
 
           index_name,
           index_type,
-          index_columns,
+          index_columns_and_opclasses,
```

Comment:
https://github.com/rails/rails/blob/5-2-stable/activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb#L464-L466

Just noticed that our ActiveRecord::ConnectionAdapters::PostgreSQL::SchemaStatements extension lives in `../connection_adapters/abstract/schema_statements.rb` instead of `../connection_adapters/postgresql/schema_statements.rb`